source.yml

```diff
@@ -73,78 +73,82 @@ categories:
-        note: "Discussed on [Episode 37 of Swift Package Indexing](https://share.transistor.fm/s/de52cb62){:target='_blank'}."
-      - identifier: danwood/SwiftUICoreImage
-        note: "Discussed on [Episode 38 of Swift Package Indexing](https://share.transistor.fm/s/f744b457){:target='_blank'}."
-  - name: December 2023
-    slug: december-2023
-    packages:
-      - identifier: pointfreeco/swift-composable-architecture
-        note: "Linked from [this Swift forums post](https://forums.swift.org/t/nominations-for-the-packages-community-showcase-on-swift-org/68168/3){:target='_blank'}."
-      - identifier: davedelong/time
-        note: "Linked from [this Swift forums post](https://forums.swift.org/t/nominations-for-the-packages-community-showcase-on-swift-org/68168/7){:target='_blank'}."
-      - identifier: PureSwift/BluetoothLinux
-        note: "Linked from [this Swift forums post](https://forums.swift.org/t/nominations-for-the-packages-community-showcase-on-swift-org/68168/4){:target='_blank'}."
-      - identifier: gohanlon/swift-memberwise-init-macro
-        note: "Discussed in [Episode 37 of Swift Package Indexing](https://share.transistor.fm/s/de52cb62){:target='_blank'}."
-      - identifier: uraimo/SwiftyGPIO
-        note: "Linked from [this Swift forums post](https://forums.swift.org/t/nominations-for-the-packages-community-showcase-on-swift-org/68168/4){:target='_blank'}."
-      - identifier: daprice/Variablur
-        note: "Linked in [Issue 637 of iOS Dev Weekly](https://iosdevweekly.com/issues/637#txS8ASr){:target='_blank'}."
-  - name: November 2023
-    slug: november-2023
-    packages:
-      - identifier: rensbreur/SwiftTUI
-        note: "Discussed on [Episode 163 of the Empower Apps Podcast](https://brightdigit.com/episodes/163-swiftly-tooling-with-pol-piella-abadia/){:target='_blank'}."
-      - identifier: migueldeicaza/SwiftGodot
-        note: "Discussed on [Season 5 Episode 28 of Compile Swift](https://www.compileswift.com/podcast/s05e28/){:target='_blank'} podcast."
-      - identifier: automerge/automerge-swift
-        note: "Discussed on [Episode 36 of Swift Package Indexing](https://swiftpackageindexing.transistor.fm/episodes/36-even-though-the-bug-is-fixed-its-not-over){:target='_blank'}."
-      - identifier: li3zhen1/Grape
-        note: "Discussed on [Episode 36 of Swift Package Indexing](https://swiftpackageindexing.transistor.fm/episodes/36-even-though-the-bug-is-fixed-its-not-over){:target='_blank'}."
-      - identifier: ActuallyTaylor/Firefly
-        note: "Discussed on [Episode 36 of Swift Package Indexing](https://swiftpackageindexing.transistor.fm/episodes/36-even-though-the-bug-is-fixed-its-not-over){:target='_blank'}."
-      - identifier: daily-co/daily-client-ios
-        note: "Discussed on [Episode 162 of the Empower Apps Podcast](https://brightdigit.com/episodes/162-building-a-video-sdk-with-marc-schwieterman/){:target='_blank'}."
+  - name: 2024
```

Comment:
Should the key here be "year" instead?

Reply:
It was when I initially wrote it, but @federicobucci had it as `name` in his sample data that I was working to so I switched it to match that. If @federicobucci agrees, I think `year` would be a more logical key name, but it would require both sides to change.

Reply:
@finestructure @daveverwer yes, I agree, `year` is a better key name!  I am going to change it in the other PR

Reply:
Done 👍